### DISCUSSION TITLE
Why @Decorators.maximizable() in Dialog.ts is not work with net8.0 ??

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#7213] [Help]
Why @Decorators.maximizable() in Dialog.ts is not work with net8.0

How do I solve the problem?

**Comment by VictorTomaili** [comment]
Previous dialogs used jQuery UI. We removed jQuery UI. The Bootstrap popups are not suitable for all use cases, so the dialogs are now opened in panel mode (maximized) by default. This decorator still exists because of backward compatibility.

**Reply by guruprasadk-reconn** [reply]
How can we make dialog to open as dialog (floating) by default instead of panel?

**Comment by nahidulislam-cse15** [comment]
How to customize Bootstrap popup size (width,height, caption)?

**Reply by arunkumar12712** [reply]
.s-Module-XXXDialog > .size {
  width: 450px;
  margin-left:600px;
 
}

.s-Module-XXXDialog  .caption{
  width: 150px;
  
}

**Reply by nahidulislam-cse15** [reply]
@arunkumar12712  not working in the new bootstrap modal UI 
![image](https://github.com/user-attachments/assets/b501cd64-6235-4f5d-ba3f-2879a650c608)

**Reply by dfaruque** [reply]
> How to customize Bootstrap popup size (width,height, caption)?

Override getDialogOptions

```
    protected getDialogOptions() {
        let opt = super.getDialogOptions();

        opt.size = 'xl';

        return opt;
    }
```

**Reply by nahidulislam-cse15** [reply]
@dfaruque  Jazakallahu khairan .It works
